server/pbench/lib/config/pbench-server.cfg.example

```diff
@@ -11,6 +11,13 @@ pbench-local-dir = /srv/pbench-local
 # Backup dir may be remote NFS mounted
 pbench-backup-dir = /mnt/pbench.archive.backup
 
+[aws-s3]
```

Comment:
Can we rename this section to be `[pbench-server-backup]`?  Technically, we are not using Amazon Web Services but any S3 provided environment, and right now only for backup purposes.

Reply:
Done